data/types/device/prime.go

```diff
@@ -21,11 +20,18 @@ type Prime struct {
 }
 
 var (
-	volumeField = types.FloatDatumField{
+	cannulaVolumeField = types.FloatDatumField{
 		DatumField:        &types.DatumField{Name: "volume"},
 		Tag:               "devicevolume",
-		Message:           "Must be greater than 0.0",
-		AllowedFloatRange: &types.AllowedFloatRange{LowerLimit: 0.0},
+		Message:           "Must be >= 0.0 and <= 3.0",
+		AllowedFloatRange: &types.AllowedFloatRange{LowerLimit: 0.0, UpperLimit: 3.0},
+	}
+
+	tubingVolumeField = types.FloatDatumField{
+		DatumField:        &types.DatumField{Name: "volume"},
+		Tag:               "devicevolume",
+		Message:           "Must be >= 0.0 and <= 100.0",
```

Comment:
We should decide whether we want these characters in the strings (that will be escaped in JSON).  Just makes it harder to read the output.